kubernetes/samples/scripts/create-weblogic-domain/model-in-image/ai-docker-file/Dockerfile

```diff
@@ -25,5 +25,6 @@ ENV AUXILIARY_IMAGE_PATH=${AUXILIARY_IMAGE_PATH}
 RUN adduser -D -u ${USERID} -G $GROUP $USER
 # ARG expansion in COPY command's --chown is available in docker version 19.03.1+.
 # For older docker versions, change the Dockerfile to use separate COPY and 'RUN chown' commands.
-COPY --chown=$USER:$GROUP ./ ${AUXILIARY_IMAGE_PATH}/
+COPY ./ ${AUXILIARY_IMAGE_PATH}/
+RUN chown -R $USER:$GROUP ${AUXILIARY_IMAGE_PATH}
```

Comment:
This looks like it reverts a 4.0 change/optimization by @ankedia ?  Did you use a recent enough of version of docker  so the docker file would work?

Reply:
I agree with Tom, I don't understand why we would split the COPY w/ chown command into two separate commands.

Reply:
fixed. I split the command since in my local box, the docker version is lower than 19.03.1.